app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :comments, class_name: 'Comments', foreign_key: 'author_id'
+  has_many :posts, class_name: 'Posts', foreign_key: 'author_id'
+  has_many :likes, class_name: 'Likes', foreign_key: 'author_id'
```

Comment:
- [ ] Here no need to define class_name here, by convention rails can easily understand the table name.
- for ex:- `has_many :likes, foreign_key: 'author_id'` here rails can understand that 
the user has many likes & it will automatically search for author_id (because of defined forieign_key otherwise it search for user_id) in Like table.
- Kindly remove `class_name: 'Comments',  class_name: 'Posts', class_name: 'Likes',`